app/controllers/manage_metadata_controller.rb

```diff
@@ -311,4 +311,12 @@ def get_page(fields)
     TemporalKeywords
     PaleoTemporalCoverages
   )
+
+  def ensure_non_nasa_draft_user
+    unless is_non_nasa_draft_user?(user: current_user, token: token)
+      clear_session_and_token_data
+
+      redirect_to root_url, flash: { error: "It appears you are not provisioned with the proper permissions to access the MMT for Non-NASA Users. Please try again or contact #{view_context.mail_to('support@earthdata.nasa.gov', 'Earthdata Support')}." }
```

Comment:
is this the exact same message as in collection_draft_proposals_controller? should this be in some sort of internationalization file so that you can reuse it? No is fine, just a suggestion for reducing duplication.

Reply:
I think that even if I tried to use the en.yml file for the error message, I'd have to have it in there twice. I think the messages are tied to controllers by action.